browser/ui/views/sidebar/sidebar_item_add_button.cc

```diff
@@ -43,22 +43,13 @@ SidebarItemAddButton::SidebarItemAddButton(
 SidebarItemAddButton::~SidebarItemAddButton() = default;
 
 void SidebarItemAddButton::OnButtonPressed() {
-  if (IsBubbleVisible())
+  if (IsBubbleVisible()) {
     return;
+  }
 
   ShowBubble();
 }
 
-void SidebarItemAddButton::OnThemeChanged() {
```

Comment:
note: we don't need to call UpdateButtonImages() manually when we use ui::ImageModel with VectorIcons.